src/components/wayback-info.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+
+/**
+ * @typedef WaybackInfo
+ * @property {Object} fromMetadata
+ * @property {Object} toMetadata
+ * @property {String} pageUrl
+ */
+
+/**
+ * Renders link to see Wayback calendar view of the page
+ * If one or both of the page diffs were sourced from Wayback, this component renders links to the Wayback source url(s)
+ *
+ * @class WaybackInfo
+ * @extends {React.Component}
+ * @param {WaybackInfoProps} props
+ */
+
+export default class WaybackInfo extends React.Component {
+  render () {
+    const waybackCalendarUrl = `https://web.archive.org/web/*/${this.props.pageUrl}`;
+    const pageUrl = <a href={waybackCalendarUrl} target="_blank" rel="noopener noreferrer">See Wayback calendar view</a>;
+    const toUrl = this.props.toMetadata.view_url ? <span> | <a href={this.props.toMetadata.view_url} target="_blank" rel="noopener noreferrer">See Wayback previous page version</a></span> : '';
+    const fromUrl = this.props.fromMetadata.view_url ? <span> | <a href={this.props.fromMetadata.view_url} target="_blank" rel="noopener noreferrer">See Wayback next page version</a></span> : '';
+    
+    return [pageUrl, toUrl, fromUrl];
```

Comment:
I think it might make sense to wrap this in an actual `<div>` to group things together, rather than simply returning a fragment. Then we can do a little more with styling in the future.

Reply:
Yeah I was wondering about the styling because personally I don't think the bare blue links look that great but was trying to be consistent with the rest of the page. Is there a designer working on this or is this the developer's responsibility? I was thinking of making them bold black links at least for the interim if that's ok.

Reply:
No designer at the moment. There’s plenty that could probably be improved with some dedicated design-related effort!

> I was thinking of making them bold black links at least in the interim if that's ok.

That sounds alright.